src/main/webapp/script.js

```diff
@@ -71,22 +87,66 @@ function passQuery() {
     });
 }
 
-function displayAmChartsMap(data, description, location, isCountyQuery) {
-  am4core.useTheme(am4themes_animated);
-  const chart = am4core.create('map', am4maps.MapChart);
-  chart.height = 550;
-  chart.zoomControl = new am4maps.ZoomControl();
-  // only allow zooming with buttons
-  chart.mouseWheelBehavior = 'none';
-
-  // Create map instance
+function getGeoData(location, isCountyQuery) {
   if (isCountyQuery) {
-    chart.geodata = location ===
+    return location ===
         '06' ? am4geodata_region_usa_caLow : am4geodata_region_usa_njLow;
   } else {
-    chart.geodata = am4geodata_usaLow;
+    return am4geodata_usaLow;
+  }
+}
+
+
+function displayVisualization(censusDataArray, description,
+  location, isCountyQuery) {
+  const geoData = getGeoData(location, isCountyQuery);
+  if (isCountyQuery) {
+    setStyleForCountyQuery();
+    const mapsData = getMapsData(censusDataArray);
+    const amChartsData = createDataArray(censusDataArray, isCountyQuery);
+    displayAmChartsMap(amChartsData, description, geoData);
+    displayCountyGeoJson(mapsData, location);
+  } else {
+    setStyleForStateQuery();
+    const amChartsData = createDataArray(censusDataArray, isCountyQuery);
+    displayAmChartsMap(amChartsData, description, geoData);
```

Comment:
Do we not want to display geocharts for this option?

Reply:
The three of us never discussed using geoJson for states because Google Maps has states by default. I will ask tomorrow if we want to also add a toggle function for a state query.